uaclient/entitlements/fips.py

```diff
@@ -561,3 +566,19 @@ def _perform_enable(self, silent: bool = False) -> bool:
             return True
 
         return False
+
+
+class FIPSPreviewEntitlement(FIPSEntitlement):
+    name = "fips-preview"
+    title = messages.FIPS_PREVIEW_TITLE
+    description = messages.FIPS_PREVIEW_DESCRIPTION
+    help_text = messages.FIPS_PREVIEW_HELP_TEXT
+    pre_enable_msg = messages.PROMPT_FIPS_PREVIEW_PRE_ENABLE
```

Comment:
I think we can set a line here for the `origin` from looking at https://esm.ubuntu.com/fips-preview/ubuntu/dists/focal/Release
```
origin = "UbuntuFIPSPreview"
```